homeassistant/components/denonavr/media_player.py

```diff
@@ -90,7 +95,16 @@ async def async_setup_entry(
         else:
             unique_id = f"{config_entry.entry_id}-{receiver_zone.zone}"
         await receiver_zone.async_setup()
-        entities.append(DenonDevice(receiver_zone, unique_id, config_entry))
+        entities.append(
+            DenonDevice(
+                receiver_zone,
+                unique_id,
+                config_entry,
+                hass.data[DOMAIN][config_entry.entry_id].get(
+                    CONF_UPDATE_AUDYSSEY, DEFAULT_UPDATE_AUDYSSEY
+                ),
```

Comment:
```suggestion
                update_audyssey
```

This would be a bit easier to follow if we did something like this above

```python
data = hass.data[DOMAIN][config_entry.entry_id]
receiver = data[CONF_RECEIVER]
update_audyssey = data.get(CONF_UPDATE_AUDYSSEY, DEFAULT_UPDATE_AUDYSSEY)
```

Reply:
Done